src/lib/Libwin/passwd.c

```diff
@@ -649,15 +663,17 @@ GetComputerDomainName(char domain_name[PBS_MAXHOSTNAME+1])
 
 	strcpy(local_name, "");
 	local_sz = sizeof(local_name);
-	GetComputerName(local_name, &local_sz);
+	if (GetComputerName(local_name, &local_sz) == 0) {
+		log_err(-1, __func__, "failed in GetComputerName");
+	}
 	strncpy(domain_name, local_name, PBS_MAXHOSTNAME);
 
 	ZeroMemory(&obj_attrs, sizeof(obj_attrs));
-	if (LsaOpenPolicy(NULL, &obj_attrs, POLICY_VIEW_LOCAL_INFORMATION,
-	&h_policy) \
-                                                        != ERROR_SUCCESS )
+	lsa_stat = LsaOpenPolicy(NULL, &obj_attrs, POLICY_VIEW_LOCAL_INFORMATION, &h_policy);
+	if ( lsa_stat != ERROR_SUCCESS ) {
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "failed in LsaOpenPolicy, with errno %d", LsaNtStatusToWinError(lsa_stat));
```

Comment:
My opinion will be to log the actual LsaNtStatus error, because 

> The return value is the Windows error code that corresponds to the Status parameter. If there is no corresponding Windows error code, the return value is ERROR_MR_MID_NOT_FOUND.

I thought of converting it to Windows error code and log the actual LsaNtStatus error code too, but thought that may confuse the user as there will be 2 error code.

Reply:
thanks, I think printing LsaNtStatus will make it more clear, rather than converting.